### PR TITLE
Quote branch

### PR DESCRIPTION
**Pull request by gorrelljd21**
Completed for task  "Add a GraphQL Schema for Quotes API#237"

Created a schema for the quotes api through initializing gqlgen and ensured that by regenerating, the models and resolvers automagically updated.

This PR includes both the schema and the queries tickets.

### CODE REVIEW

**Review by boggsey — APPROVED** [review]
Good work! Now you'll need to write your resolvers!

File: main.go [hunk 1]
```diff
@@ -32,11 +32,11 @@ func main() {
...
-	r.GET("/quotes", getRandomQuoteSQL)
-	r.GET("/quotes/:id", getQuoteByIdSQL)
-	r.POST("/quotes", addQuoteSQL)
-	r.DELETE("/quotes/:id", deleteQuote)
-	r.Run("0.0.0.0:8080")
+	r.GET("/quote", getRandomQuoteSQL)
+	r.GET("/quote/:id", getQuoteByIdSQL)
+	r.POST("/quote", addQuoteSQL)
+	r.DELETE("/quote/:id", deleteQuote)
+	r.Run("0.0.0.0:8081")
```
Comment (librairica) [inline main.go]: I had to change this to 8080 to run both your API servers
```suggestion
	r.Run("0.0.0.0:8080")
```

**Review by librairica — CHANGES_REQUESTED** [review]
When I tried to test your code, I ran into some port in use errors, so I changed your rest api's port to 8080 and I could at least query in the GraphiQL explorer. I was getting empty strings back on all quote fields in GraphiQL, so I went ahead and checked the REST API in postman and it was also returning a random quote with empty fields. Is the db empty? It would be super helpful if you updated the description for this PR with some instructions for how to test.

**Review by librairica — APPROVED** [review]
Once I added your db values to my zsh, both queries worked!

**Pull request merged**